src/components/item.jsx

```diff
@@ -24,13 +27,27 @@ class Item extends React.Component {
     this.setState({
       itemId: this.props.itemId
     });
-    this.getItem();
+    // this.getItem();
     this.getAvailability();
     this.getReviews();
   }
 
   getItem() {
-    // This will call out to Item service
+    let itemId = this.state.itemId;
+    $.ajax({
+      method: 'GET',
+      url: itemDetailsUrl + 'single/' + itemId,
+      datatype: 'json',
+      success: (data) => {
+        console.log('game data received', data);
+        this.setState({
+          //
+        })
+      },
+      error: (err) => {
+        console.log('err', err);
+      }
+    })
 
     // Fake data generator in case Item service unresponsive
```

Comment:
Comment also applies to lines 58 and 64. Just curious about your plan with fallbacks; do you intend to dynamically render fake data depending on bundleId? I ask because my plan was just to have a static datum if there wasn't data that would be the same on each bundle page (i.e., without changing).

Reply:
Probably just static data, but I might get fancy with some simple buckets.